src/lightning/__setup__.py

```diff
@@ -31,11 +31,15 @@ def _adjust_manifest(**kwargs: Any) -> None:
     with open(manifest_path) as fp:
         lines = [ln.rstrip() for ln in fp.readlines()]
     lines += [
+        "include requirements/base.txt",
         "recursive-include src/lightning *.md",
-        "recursive-include requirements *.txt",
         "recursive-include src/lightning/app/ui *",
         "recursive-include src/lightning/cli/*-template *",  # Add templates as build-in
         # fixme: this is strange, this shall work with setup find package - include
+        "prune requirements/app",
+        "prune requirements/lite",
+        "prune requirements/pytorch",
+        "prune requirements/lit",
```

Comment:
@carmocca I think this is still needed or if you can integrate it in #15549

Reply:
How can I tell if it's needed?

Reply:
when you create package and these files are still included but they are unrelated so shall not be there :)

Reply:
Got it. Done in https://github.com/Lightning-AI/lightning/pull/15646/commits/cf6a95e01fe2c428cc9b36bf35a8fd4b10629957